x-pack/plugins/maps/public/classes/sources/es_search_source/scaling_form.tsx

```diff
@@ -177,9 +215,49 @@ export class ScalingForm extends Component<Props, State> {
     );
   }
 
+  _renderMVTRadio() {
+    const labelText = i18n.translate('xpack.maps.source.esSearch.useMVTVectorTiles', {
+      defaultMessage: '.mvt tile protocol',
+    });
+    const label = (
+      <>
+        {labelText}
+        <EuiBetaBadge
```

Comment:
This beta badge is making the option render funny. The text is lower then it should be and not centered.

<img width="200" alt="Screen Shot 2020-08-17 at 8 01 09 AM" src="https://user-images.githubusercontent.com/373691/90404602-d8adb880-e05f-11ea-90f2-63edec574c33.png">